modules/util/portal-tools-rest-builder/src/test/resources/com/liferay/portal/tools/rest/builder/dependencies/rest-openapi.yaml

```diff
@@ -76,6 +76,26 @@ components:
                     type: array
             title: TestArray
             type: object
+        TestWithJsonMapAttribute:
```

Comment:
Maybe better using a name that matches the previous format, that means without the "With" word: `TestJsonMapAttribute` (it needs to be sorted then)

Reply:
Yeah sure thing. I will change it right now.